src/components/view/ContactInfo/index.js

```diff
@@ -20,8 +21,9 @@ class ContactInfo extends Component {
                 <pre>UID from URL: <span style={{color:'red'}}>{uid}</span></pre>
 
                 <div>
-                    <ContactProfileCard titleColor={palette.primary1Color} contact={contact} />
-                    <ContactNotesCard titleColor={palette.primary1Color} />
+                    <ContactInfoProfileCard titleColor={palette.primary1Color} contact={contact} />
```

Comment:
Why not just pass in the palette and reduce the API memorization needs? While this is a solid and viable solution, if you pass in just the palette, then the only API to learn is the primary palette (instead of also remembering `titleColor`. 
#question

Reply:
I felt that it was unnecessary to pass through a whole bunch of data via props that the underlying component never utilizes. Doing what you mentioned would be better in case we need to extend upon this component in the future though, and I suppose it would also be nice to standardize that pattern across all of our components. So I'll make that change